CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ if(MSVC)
         PATH=$ENV{PATH}
         PYTHONPATH=${ROOTTEST_ENV_PYTHONPATH}:$ENV{PYTHONPATH})
   endif()
+  set(PYTHON_EXECUTABLE_Development_Main ${PYTHON_EXECUTABLE})
```

Comment:
This overriding is not correct, for the reasons I explained in my other comment about `${PYTHON_EXECUTABLE_Development_Main}`.

Reply:
@etejedor OK, thanks, so I will try to debug CMake to understand why PYTHON_EXECUTABLE_Development_Main is not properly set...

Reply:
@etejedor roottest.git missing all CMake part connected with standalone discovery of python.

Reply:
@etejedor @oshadura If you agree, I'll make a temporary fix, until the standalone version is properly fixed